planner/core/memtable_predicate_extractor.go

```diff
@@ -589,6 +589,9 @@ func (e *ClusterLogTableExtractor) Extract(
 	}
 
 	remained, startTime, endTime := e.extractTimeRange(ctx, schema, names, remained, "time", time.Local)
+	// The time unit for search log is millisecond.
+	startTime = startTime / int64(time.Millisecond)
+	endTime = endTime / int64(time.Millisecond)
```

Comment:
Why change to `millisecond` here? Can we use `nanosecond` here?

Reply:
As the comment said in line #592, The time unit for the search log is millisecond. If we don't change in here, we should change in the other place.